src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -307,7 +305,7 @@ public int compareTo(CommentAttributes o) {
         return o.createdAt.compareTo(createdAt);
     }
 
-    private String getEditedAtText(Boolean isGiverAnonymous, String displayGiverAs,
+    public String getEditedAtText(Boolean isGiverAnonymous, String displayGiverAs,
             String displayTimeAs) {
```

Comment:
Since the time format is now standardized, you can remove this parameter and make the formatting part of the method instead.